Model/Animal.cs

```diff
@@ -0,0 +1,73 @@
+﻿    public class Animal
+    {
+    internal const ushort MAX_AGE = 2000;
+    internal const float MIN_SIZE = 0.03f;
+    internal const float MAX_SIZE = 33.0f;
+    internal const float MAX_JUMP_LENGTH = 5.25f;
+    internal  const byte MAX_FROG_AGE = 17;
+    internal  const float MIN_FROG_SIZE = 0.008f;
+    internal const float MAX_FROG_SIZE = 0.07f;
+    internal const byte MAX_CAT_AGE = 43;
+    internal const float MIN_CAT_SIZE = 0.08f;
+    internal const float MAX_CAT_SIZE = 0.55f;
+    internal string _name="default name";
+    internal ushort _age=0;
+    internal float _size= MIN_SIZE;
+    public Animal()
+    {
+    }
+    public Animal(string name)
+    {
+        Name = name;
+    }
+    public Animal(string name, ushort age, float size)
+    {
+        Name = name;
+        Age = age;
+        Size = size;
+    }
+    //properties
+    public string Name
+        {
+            get { return _name; }
+            set { if( _name != null)
+                _name = value;
+                else throw new Exception("Значение поля _name не может быть пустым!");
+            }
+        }
+    virtual public ushort Age
+        {
+            get { return _age; }
+            set
+            {
+                if (value >= 0 && value <= MAX_AGE)
+                    _age = value;
+                else throw new Exception($"Значение поля _age должно иметь значение от 0 до {MAX_AGE}.");
+            }
+        }
+    virtual public float Size
+        {
+            get { return _size; }
+            set
+            {
+                if (value >= MIN_SIZE && value <= MAX_SIZE)
+                    _size = value;
+                else throw new Exception($"Значение поля _size должно иметь значение от {MIN_SIZE} до {MAX_SIZE}.");
+            }
+        }
+    //methods
+    public virtual void PrintInfo ()
+    {
+        Console.WriteLine($"Информация о животном:\n Имя: {Name}. Возраст: {Age} лет. Размер: {Size}м.");
```

Comment:
note: выводить свойства или поля?
```suggestion
        Console.WriteLine($"Информация о животном:\n Имя: {_name}. Возраст: {Age} лет. Размер: {Size}м.");
```
единого мненения нет, зависит от ситуации